release-notes.md

```diff
@@ -1,6 +1,7 @@
 ## 2.0.0
 
-* User Story 132316 : New 'Performance data' section added to task list.
```

Comment:
You've removed a line from the release notes, was that intended?

Reply:
ill put that back